### DISCUSSION TITLE
I inserted the dimension 1024 vector successfully, but when I searched it, the results were not accurate at all

### DISCUSSION THREAD

**Discussion by zhangnn520** [milvus-io/milvus#29134] [Q&A and General discussion]
I inserted the dimension 1024 vector successfully, but when I searched it, the results were not accurate at all.Has anyone seen the same problem?why?how to do deal it?

**Comment by zhangnn520** [comment]
milvus 1.1.1-cpu-d061621-330cc6
我用的是这个镜像

**Comment by yhmo** [comment]
ANN search is approximate search, the result depends on the embedding model and the index/search parameters. Sometimes it cannot ensure that "the top1 result is the best result that you are looking for", the best result might be at the top2 or top3...

For IVF index, when you create index, a parameter "nlist" is assigned. When you search, a parameter "nprobe" could affect the result. With smaller nprobe, the result is worse. With larger nprobe, the result is better. If you set nprobe=nlist value, then the result is the best.

**Reply by zhangnn520** [reply]
thank your reply.But when I use low latitudes such as 256 dimensions, I can calculate the result normally.High latitude but display error, their corresponding parameters, I also adjusted, useless

**Reply by yhmo** [reply]
The latest milvus version is preferred: https://milvus.io/docs/install_standalone-docker.md with the latest sdk: https://milvus.io/docs/install-pymilvus.md

Not sure what you mean "High latitude but display error". Show me the parameters: dimension, row count, index parameters, search parameters, the results, etc.

**Reply by zhangnn520** [reply]
High dimension
